.github/workflows/docs-latest-public-wrapper.yaml

```diff
@@ -2,16 +2,21 @@ name: "[post-commit] Docs build and deploy to GitHub pages on main"
 
 on:
   workflow_dispatch:
-
 jobs:
-  build-docker-artifact:
-    uses: ./.github/workflows/build-docker-artifact.yaml
-    secrets: inherit
+  # build-docker-artifact:
```

Comment:
The changes to this file are purely for testing purposes. There should be no difference besides the release section at the end.